### PR TITLE
New function for GraphScene: area between two curves

### PR DESCRIPTION
**Pull request by kolibril13**
This new function can be tested with:
```python
class area(GraphScene):
    CONFIG = {
        "x_min" : 0,
        "x_max" : 5,
        "y_min" : 0,
        "y_max" : 6,
        "y_tick_frequency" : 1,
        "x_tick_frequency" : 1,
        "x_labeled_nums" : [0,2,3]
    }
    def construct(self):
        self.setup_axes(animate=False)
        curve1 = self.get_graph(lambda x : 4*x-x**2, x_min=0,x_max=4)
        curve2 = self.get_graph(lambda x : 0.8*x**2-3*x+4, x_min=0,x_max=4)
        line1 = self.get_vertical_line_to_graph(2,curve1,DashedLine,color=YELLOW)
        line2 = self.get_vertical_line_to_graph(3,curve1,DashedLine,color=YELLOW)

        area = self.get_area(curve2,2,3,bounded=curve1)

        self.play(ShowCreation(curve1), ShowCreation(curve2),
                  ShowCreation(line1), ShowCreation(line2))
        self.play(ShowCreation(area))
        self.wait()
```
area

### CODE REVIEW

File: manim/scene/graph_scene.py [hunk 1]
```diff
@@ -101,7 +101,7 @@ def setup_axes(self, animate=False):
             color=self.axes_color,
             include_tip=self.include_tip,
             add_start=self.x_add_start,
-            add_end=self.x_add_end,
+            add_end=self.x_add_end
```
Comment (leotrs) [inline manim/scene/graph_scene.py:104]: Question: did you do this manually or did `black` do this? It is my understanding that black prefers trailing commas in these cases. Please make sure to run `black` locally before submitting a PR.

**Comment by leotrs** [comment]
@kolibril13 , another great addition to manim! Please make sure to run black, and to resolve the merge conflicts, before this can be merged.

File: manim/scene/graph_scene.py (hunk 1 above)
Comment (kolibril13) [reply manim/scene/graph_scene.py]: I did this manually.
The next commit will be formated

File: manim/scene/graph_scene.py [hunk 2]
```diff
@@ -101,7 +101,7 @@ def setup_axes(self, animate=False):
             color=self.axes_color,
             include_tip=self.include_tip,
             add_start=self.x_add_start,
-            add_end=self.x_add_end
+            add_end=self.x_add_end,
```
Comment (kolibril13) [inline manim/scene/graph_scene.py]: It seems like black wants these commas.
I don't like them too much, but I can live with it.

Comment (leotrs) [reply manim/scene/graph_scene.py]: Embrace the way of black! ;)

File: manim/scene/graph_scene.py [hunk 3]
```diff
@@ -515,11 +515,18 @@ def get_riemann_rectangles(
...
+                    map(
+                        VectorizedPoint,
+                        [
+                            self.coords_to_point(x, y_point),
+                            self.coords_to_point(x + width_scale_factor * dx, y_point),
+                            graph_point,
+                        ],
+                    )
+                )
+            )
```
Comment (leotrs) [inline manim/scene/graph_scene.py]: Ok I can take the commas, but this is pretty ridiculous to me. @eulertour your thoughts? Can we make black behave here?

**Pull request merged**